composer.json

```diff
@@ -10,6 +10,6 @@
         "php": ">=7.0",
         "adci/full-name-parser": "^0.2.4",
         "ramsey/uuid": "^3.7",
-        "swedbank-pay/swedbank-pay-woocommerce-core": "^1.0"
+        "swedbank-pay/swedbank-pay-woocommerce-core": "dev-master"
```

Comment:
Isn't it better to just add a new tag in `swedbank-pay-woocommerce-core`?

Reply:
@asbjornu Excuse me, we need it at the moment. We will bump a new verstion tag when will release a stable version of woocommerce plugin. The core library can be changed in any moment now

Reply:
Ok.